File: src/main/java/io/github/mschout/gitlab/toggltimer/GitlabTogglTimerApplication.java
package io.github.mschout.gitlab.toggltimer;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.web.client.RestTemplateBuilder;
import org.springframework.context.annotation.Bean;
import org.springframework.web.client.RestTemplate;

@SpringBootApplication
public class GitlabTogglTimerApplication {

  public static void main(String[] args) {
    SpringApplication.run(GitlabTogglTimerApplication.class, args);
  }

  @Bean
  public RestTemplate restTemplate(RestTemplateBuilder builder) {
    return builder.build();
  }
}

File: src/main/kotlin/io/github/mschout/gitlab/toggltimer/GitlabTogglTimerApplication.kt
package io.github.mschout.gitlab.toggltimer

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.restclient.RestTemplateBuilder
import org.springframework.context.annotation.Bean
import org.springframework.web.client.RestTemplate

@SpringBootApplication
class GitlabTogglTimerApplication {
  @Bean fun restTemplate(builder: RestTemplateBuilder): RestTemplate = builder.build()
}

fun main(args: Array<String>) {
  SpringApplication.run(GitlabTogglTimerApplication::class.java, *args)
}
